listenbrainz/webserver/templates/navbar.html

```diff
@@ -19,8 +19,8 @@
       <ul class="nav navbar-nav">
         {% if not hide_navbar_user_menu %}
           {% if current_user.is_authenticated %}
-            <li><a href="{{ url_for('user.profile', user_name=current_user.musicbrainz_id) }}">My Listens</a></li>
-            <li><a href="{{ url_for('index.feed') }}">My Feed</a></li>
+            <li><a href="{{ url_for('user.profile', user_name=current_user.musicbrainz_id) }}">Examine</a></li>
+            <li><a href="{{ url_for('index.feed') }}">Feed</a></li>
```

Comment:
We discussed this [on IRC](https://chatlogs.metabrainz.org/libera/metabrainz/2022-02-14/?msg=4953333&page=1) and decided to leave it as-is for now until we redesign the navigation, to avoid introducing to big  navigation changes in a row.

We also discussed reorganizing elements in the navbar, but probably not worth it for now
![image](https://user-images.githubusercontent.com/6179856/153855871-8f94b74c-f48d-4e6a-86b9-50c5c608045f.png)

Reply:
More talking, and we decided to go for "Home" and "Feed".
Both are standard nav item names and widely understood to relate to the current user